test/libyul/objectCompiler/eof/creation_with_immutables.yul

```diff
@@ -0,0 +1,78 @@
+object "a" {
+    code {
+        mstore(0, eofcreate("b", 0, 0, 0, 0))
+        return(0, 32)
+    }
+
+    object "b" {
+        code {
+            auxdatastore(0, 0, 0x1122334455667788990011223344556677889900112233445566778899001122)
+            auxdatastore(0, 32, 0x1234567890123456789012345678901234567890123456789012345678901234)
```

Comment:
Test needs updating. We no longer have this.

Reply:
right. Done